.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
```

Comment:
>If you set blank_issues_enabled to true, people will have the option to open a blank issue.

we should encourage the contributors to use issue template?

Reply:
Yes, but I think there is no reason to restrict this now. I'm ok with any setting here.

Reply:
I think `blank_issues_enabled: false` is more reasonable

Reply:
```suggestion
blank_issues_enabled: false
```